test/src/JsonmlToReact.spec.js

```diff
@@ -158,7 +158,12 @@ describe('JsonmlToReact class', function () {
 
       jsonmlToReact.convert(node);
 
-      expect(spy.calledWith('p', { key: 0 }, 'i am a text node')).to.be.true;
+      expect(spy.calledWith('p', {
+        className: undefined,
+        class: undefined,
+        key: 0,
+        style: undefined
+      }, 'i am a text node')).to.be.true;
```

Comment:
trivial - might be nice to assign the props object to a variable

Reply:
minor - I think we should add a test that verifies that none of these new props are sent to a `converter`?